src/SearchableEntity.php

```diff
@@ -47,7 +50,7 @@ public function __construct(
         $this->entity = $entity;
         $this->entityMetadata = $entityMetadata;
         $this->normalizer = $normalizer;
-        $this->useSerializerGroups = isset($extra['useSerializerGroup']) && $extra['useSerializerGroup'];
+        $this->normalizationGroups = $extra['normalizationGroups'] ?? [];
```

Comment:
this could be some sort of a BC break, but will leave it to you to decide if we need a compatibility here

Reply:
I don't think we need to keep compatibility, but is the change just from `useSerializerGroup` to `normalizationGroups`? and both are arrays right?

Reply:
`useSerializerGroup` was a boolean